### PR TITLE
Remove Rafter from Compass Runtime Agent

### PR DESCRIPTION
**Pull request by mvshao**
**Description**
The feature will be no longer used in Kyma 2.1, so we need to drop the functionality.

Changes proposed in this pull request:

- Delete all Rafter dependencies

**Related issue(s)**
#12844

### CODE REVIEW

File: components/compass-runtime-agent/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The Runtime Agent uses these environment variables:
 - **APP_CA_CERTIFICATES_SECRET** specifies the Namespace and the Name of the Secret in which to store the CA certificate.
 - **APP_SKIP_COMPASS_TLS_VERIFY** specifies whether to communicate with the Connector Service and the Director with disabled TLS verification.
 - **APP_GATEWAY_PORT** specifies the Application Gateway port.
-- **APP_UPLOAD_SERVICE_URL** specifies the URL of the upload service.
+- **APP_UPLOAD_SERVICE_URL** specifies the URL of the upload service. **(_deprecated_)**
```
Comment (akgalwas) [inline components/compass-runtime-agent/README.md:36]: I would remove this.

File: components/compass-runtime-agent/cmd/init.go [hunk 2]
```diff
@@ -11,11 +11,8 @@ import (
 	appclient "github.com/kyma-project/kyma/components/application-operator/pkg/client/clientset/versioned"
 	"github.com/kyma-project/kyma/components/compass-runtime-agent/internal/apperrors"
 	"github.com/kyma-project/kyma/components/compass-runtime-agent/internal/kyma"
-	"github.com/kyma-project/kyma/components/compass-runtime-agent/internal/kyma/apiresources/rafter"
```
[the remark this answers is not in the document]
Comment (mvshao) [reply components/compass-runtime-agent/cmd/init.go:14]: This dependency is already deleted from the file.

**Review by majakurcius — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Review by franpog859 — APPROVED** [review]

**Pull request merged**